### PR TITLE
folder structure & its path are changed

### PR DESCRIPTION
**Pull request by sathyasub**

### CODE REVIEW

**Comment by MaathavanJkr** [comment]
Thank you for the pull. But Better ask and assign your self for the issue first. So the work wont get duplicated. :)

**Comment by MaathavanJkr** [comment]
@sathyasub Have you tested the code before submitting? Its not working for me and index.html should be on root folder not in web.

**Comment by sathyasub** [comment]
@MaathavanJkr can you review the last commit

File: 404.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
             return check;
         };
         if (mobCheck()) {
-            window.location = "mobile.html"
+            window.location = "mobile/mobile.html"
         } 
         else {
             window.location = "web.html"
```
Comment (MaathavanJkr) [inline 404.html:19]: Should be changed to web/

File: web/web.html [hunk 2]
```diff
@@ -5,16 +5,16 @@
 <head>
     <title>வரைபடவி - Varaipadavi</title>
 
-    <link rel="icon" href="img/favicon.png" type="image/png">
+    <link rel="icon" href="assets/favicon.png" type="image/png">
```
Comment (MaathavanJkr) [inline web/web.html:8]: ../

File: web/w-study.html [hunk 3]
```diff
@@ -102,7 +102,7 @@
             </div>
 
             <div id="map-area" class="d-flex justify-content-center">
-                <img src='img/lkmap.png' usemap='#lkmap' class='map img-fluid' title='imgmap201293016112'
+                <img src='..assets/img/lkmap.png' usemap='#lkmap' class='map img-fluid' title='imgmap201293016112'
```
Comment (MaathavanJkr) [inline web/w-study.html:105]: missing /

File: web/study.html [hunk 4]
```diff
@@ -51,15 +51,15 @@
                     <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-home"></i>Home</a></li>
                     <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-globe"></i>About us</a>
                     </li>
-                    <li class="nav-item"><a class="nav-link" href="mobile.html"><i
+                    <li class="nav-item"><a class="nav-link" href="mobile/mobile.html"><i
                                 class="pr-1 fa fa-flag"></i>Quiz</a></li>
-                    <li class="nav-item"><a class="nav-link" href="rank.html"><i class="pr-1 fa fa-list-ol"></i>Rank</a>
+                    <li class="nav-item"><a class="nav-link" href="web/rank.html"><i class="pr-1 fa fa-list-ol"></i>Rank</a>
```
Comment (MaathavanJkr) [inline web/study.html:56]: rank.html should be in mobile

File: web/study.html [hunk 5]
```diff
@@ -51,15 +51,15 @@
...
                     </li>
-                    <li class="nav-item active"><a class="nav-link" href="study.html"><i
+                    <li class="nav-item active"><a class="nav-link" href="web/study.html"><i
```
Comment (MaathavanJkr) [inline web/study.html:58]: should be in mobile

**Review by MaathavanJkr — CHANGES_REQUESTED** [review]
and make the web.html and mobile.html as index.html as the path differentiate both. And make sure you put the files on the correct folder again as rank.html and study.html should be on mobile folder.

**Comment by MaathavanJkr** [comment]
@sathyasub and also there was another file added called .DS_Store . remove that too

File: web/study.html (hunk 4 above)
Comment (sathyasub) [reply web/study.html]: updated

File: web/w-study.html (hunk 3 above)
Comment (sathyasub) [reply web/w-study.html]: updated

File: web/web.html (hunk 2 above)
Comment (sathyasub) [reply web/web.html]: updated

**Comment by sathyasub** [comment]
@MaathavanJkr all the changes reported are added.. Please review it

**Comment by MaathavanJkr** [comment]
Nice work. :) Thank You. ;)

**Comment by MaathavanJkr** [comment]
@sathyasub oops sorry i merged the pull. but saw the issue now. Login Signup and forgot html files also should be in the root folder. Can you make those changes too?

**Pull request merged**